resources/js/components/Chart.tsx

```diff
@@ -0,0 +1,90 @@
+import "chart.js/auto";
+import React from "react";
+import { Bar } from "react-chartjs-2";
+
+interface DataSetInterface {
+	label: string;
+	data: number[];
+	backgroundColor: string;
+}
+
+export interface ChartInterface {
+	labels: string[];
+	dataSets: DataSetInterface[];
+	dataType: "NUMBER" | "CURRENCY";
```

Comment:
Is this supposed to hold "CURRENCY" as a dataType? I doubt it will ever be used.